wgsl/index.bs

```diff
@@ -5207,11 +5166,11 @@ If any of the parameters are out of bounds, then zero in all components.
 Writes a single texel to a texture.
 
 ```rust
-textureStore(t : texture_storage_wo_1d<F>, coords : i32, value : vec4<T>) -> void
-textureStore(t : texture_storage_wo_1d_array<F>, coords : i32, array_index : i32, value : vec4<T>) -> void
-textureStore(t : texture_storage_wo_2d<F>, coords : vec2<i32>, value : vec4<T>) -> void
-textureStore(t : texture_storage_wo_2d_array<F>, coords : vec2<i32>, array_index : i32, value : vec4<T>) -> void
-textureStore(t : texture_storage_wo_3d<F>, coords : vec3<i32>, value : vec4<T>) -> void
+textureStore(t : access(write) texture_storage_1d<F>, coords : i32, value : vec4<T>) -> void
```

Comment:
I wonder if we should use the proper decoration here, like `[[access(write)]]`

Reply:
Done.